prog/vm/github_runner.rb

```diff
@@ -223,6 +223,39 @@ def setup_info
       UBICLOUD_CACHE_URL=#{Config.base_url}/runtime/github/" | sudo tee -a /etc/environment
     COMMAND
 
+    if vm.vm_host.id == Vm[Config.docker_mirror_server_vm_id]&.vm_host&.id
+      mirror_server_address = "mirror.ubicloudregistry.com"
```

Comment:
Why don’t we use an existing domain? Since we don't own `ubicloudregistry.com` domain, I prefer to use a subdomain from `ubicloud.com`